website/versioned_docs/version-0.2.x/client-setup.mdx

```diff
@@ -45,7 +49,10 @@ composedb composite:compile my-first-composite.json src/__generated__/definition
 
 ## Configuring the client
 
-Once the composite definition file is written, for example in `src/__generated__/definition.js` using the example above, it can be imported in the app to configure the client:
+Once the composite definition file is written, for example in `src/__generated__/definition.js` using the example above, it can be imported in the app to configure the client.ceramicnetwork.
+
+Make sure you have the following packages installed, before running this code:
+ * `@composedb/client`
```

Comment:
This is part of the installation step

Reply:
Maybe we could link to the installation page here instead? To make sure people don't miss installing the packages.